pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -171,6 +173,35 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 			return reconcile.Result{}, nil
 		}
 
+		// ceph crash prune configuration
+		if cephCluster.Spec.CrashCollector.DaysToRetain < 1 {
+			logger.Debugf("Deleting cronjob if it exists...")
+			cronJob := &v1beta1.CronJob{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      PrunerName,
+					Namespace: namespace,
+				},
+			}
+			logger.Debugf("Deleting cronjob if it exists...")
+			err := r.client.Delete(context.TODO(), cronJob)
+			if err == nil {
```

Comment:
Why do we ignore the error?

Reply:
Following @travisn 's suggestion above to just delete the cronjob and ignore errors if the cronjob isn't present. I don't see a way from the documentation to distinguish between errors returned by Delete, so I was assuming that if Delete fails then the cronjob doesn't exist. But thinking about it a little more, it does make more sense to check for the cron job first before deleting so that the reconciler doesn't have to keep calling Delete all the time.

Reply:
Instead of calling `Get()`, then `Delete()` if it exists, it's faster just to call `Delete()` and then ignore the failure if it's `errors.IsNotFound(err)`. If you search for that, you should see a number of examples.

Reply:
Ah just found an example of that right after I sent my comment so I deleted it an made the change you just suggested.